File: MODUL 2/C/PRAK204-2310817320006-Alya_Rosaan.c
#include <stdio.h>
#include <math.h>

int main(){
    double phi = 3.14286;
    double jari_jari, tinggi_bejana;
    double jari_jari_2, tinggi_bejana_2;

    printf("input\n");
    printf("");
    scanf("%lf", &jari_jari);
    printf("");
    scanf("%lf", &tinggi_bejana);

    double volume = (phi * pow(jari_jari, 2) * tinggi_bejana);
    double luas = (2 * phi * jari_jari * (jari_jari+tinggi_bejana));
    double keliling = (2 * phi * jari_jari);

    printf("output\n");
    printf("volume = %.2lf\n", volume);
    printf("luas = %.2lf\n", luas);
    printf("keliling = %.2lf\n\n", keliling);

    printf("input\n");
    printf("");
    scanf("%lf %lf", &jari_jari_2, &tinggi_bejana_2);

    double volume1 = phi * pow(jari_jari_2, 2) * tinggi_bejana_2;
    double luas1 = 2 * phi * jari_jari_2 * (jari_jari_2+tinggi_bejana_2);
    double keliling1 = 2 * phi * jari_jari_2;

    printf("output\n");
    printf("volume = %.2lf\n", volume1);
    printf("luas = %.2lf\n", luas1);
    printf("keliling = %.2lf", keliling1);
}

File: MODUL 2/PYTHON/PRAK204-2310817320006-Alya_Rosaan.py
def mynum1(jari_jari,tinggi_bejana):
    volume = 22/7 * pow(jari_jari, 2) * tinggi_bejana
    return volume
def mynum2(jari_jari,tinggi_bejana):
    luas = 2 * 22/7 * jari_jari * (jari_jari+tinggi_bejana)
    return luas
def mynum3(jari_jari,tinggi_bejana):
    keliling = 2 * 22/7 * jari_jari
    return keliling

#case 1
print("input")
jari_jari = float(input())
tinggi_bejana = float(input())

print("output")
hasil= mynum1(jari_jari,tinggi_bejana)
print(f"volume = {hasil:.2f}")

hasil_1= mynum2(jari_jari,tinggi_bejana)
print("luas = {:.2f}".format(hasil_1))

hasil_2= mynum3(jari_jari,tinggi_bejana)
print(f"keliling = {hasil_2:.2f}")

#case 2
print("input")
jari_jari,tinggi_bejana = map(float, input().split())

print("output")
hasil= mynum1(jari_jari,tinggi_bejana)
print(f"volume = {hasil:.2f}")

hasil_1= mynum2(jari_jari,tinggi_bejana)
print("luas = {:.2f}".format(hasil_1))

hasil_2= mynum3(jari_jari,tinggi_bejana)
print(f"keliling = {hasil_2:.2f}")



